schema.graphql

```diff
@@ -198,6 +198,8 @@ type CumulativeVolume @entity {
     type: VolumeType!
     tillTimestamp: DateTime!
     amount: BigInt!
+    locked: BigInt!
```

Comment:
Two things:

1. Having a think about the `total`, I think it better to name this `BridgeVolume` to make it totally clear.


2. Instead of adding this to the return type, the "locked" and "total" should be a `type` as in `Locked` and `Total`. So your query would look like:

```grapql
query MyQuery {
  cumulativeVolumes(where: {type_eq: Locked}) {
    amount
    type
  }
}
```

And return would be:

```json
{
  "data": {
    "cumulativeVolumes": [
      {
        "amount": "798800",
        "type": "Locked"
      },
      {
        "amount": "848725",
        "type": "Locked"
      },
      {
        "amount": "4687089",
        "type": "Locked"
      }
    ]
  }
}
```

Reply:
Makes senses, but just to clarify in this statement should "total" and 'Total' be replaced by BridgeVolume?
 
> 2. Instead of adding this to the return type, the "locked" and "total" should be a `type` as in `Locked` and >`Total`. So your query would look like:

Reply:
Yes, please.